tictactoe/eacs/src/test/kotlin/com/anaplan/engineering/azuki/tictactoe/eacs/BEH6.kt

```diff
@@ -11,7 +11,7 @@ import com.anaplan.engineering.azuki.tictactoe.dsl.TicTacToeScenario
 """)
 class BEH6 : TicTacToeScenario() {
 
-    @Eac("A player can win by completing a line of three tokens")
+    @Eac("The game is complete if a player has won")
```

Comment:
Sure, but I have no intention of doing that work, I just need to merge a minimal impl so that I can use it test persistence end-to-end within azuki

I thought the intention of your work was to integrate verification.. there's a lot more to do in this PR